app/routes.py

```diff
@@ -1,2 +1,131 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from app.models.planet import Planet
+from app.route_helpers import validate_model
 
+bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+@bp.route("", methods=["POST"])
+def create_planet():
+    request_body = request.get_json()
+    new_planet = Planet(name=request_body["name"],
+                        description=request_body['description'],
+                        radius=request_body['radius']
+                        )
```

Comment:
👀 Rather than explicitly reading the values from the requets body here, we would like to use our `from_dict` helper.

Also, note that if the body is missing any of these keys, it would result in a `KeyError`. Consider using a `try`/`except` to handle that without crashing (we could return a 400 Bad Request).